routers/api/v1/api.go

```diff
@@ -864,6 +864,22 @@ func Routes() *web.Router {
 		})
 	}
 
+	addActionsWorkflowRoutes := func(
+		m *web.Router,
+		reqChecker func(ctx *context.APIContext),
+		actw actions.WorkflowAPI,
+	) {
+		m.Group("/actions", func() {
+			m.Group("/workflows", func() {
+				m.Get("", reqToken(), reqChecker, actw.ListRepositoryWorkflows)
+				m.Get("/{workflow_id}", reqToken(), reqChecker, actw.GetWorkflow)
+				m.Put("/{workflow_id}/disable", reqToken(), reqChecker, actw.DisableWorkflow)
+				m.Post("/{workflow_id}/dispatches", reqToken(), reqChecker, bind(api.CreateActionWorkflowDispatch{}), actw.DispatchWorkflow)
+				m.Put("/{workflow_id}/enable", reqToken(), reqChecker, actw.EnableWorkflow)
+			}, context.ReferencesGitRepo(), reqRepoWriter(unit.TypeActions))
```

Comment:
`GET` methods just need `reqRepoReader(unit.TypeActions)`, `POST` and `PUT` needs `reqRepoWriter(unit.TypeActions)`.